pkg/source/source.go

```diff
@@ -86,8 +86,8 @@ func (ks *Kind) Start(handler handler.EventHandler, queue workqueue.RateLimiting
 	i, err := ks.cache.GetInformer(ks.Type)
 	if err != nil {
 		if kindMatchErr, ok := err.(*meta.NoKindMatchError); ok {
-			log.Error(err, "if %s is a CRD, should install it before calling Start",
-				kindMatchErr.GroupKind)
+			log.Error(err, fmt.Sprintf("if %s is a CRD, should install it before calling Start",
```

Comment:
For better or for worse, this is specifically discouraged by logr. https://github.com/go-logr/logr/blob/9fb12b3b21c5415d16ac18dc5cd42c1cfdd40c4e/logr.go#L11

logr wants you to use structured logging.

Reply:
fixed, thanks.